app/src/main/java/com/cornellappdev/uplift/ui/components/home/HomeCard.kt

```diff
@@ -103,12 +99,30 @@ fun HomeCard(gym: UpliftGym, onClick: () -> Unit) {
                                 contentDescription = "Dumbbell",
                                 colorFilter = ColorFilter.tint(color = GRAY04)
                             )
-                            Spacer(Modifier.width(4.dp))
-                            Image(
-                                painter = painterResource(id = R.drawable.ic_bowling_pins),
-                                contentDescription = "Bowling Pins",
-                                colorFilter = ColorFilter.tint(color = GRAY04)
-                            )
+                            if (gym.bowlingInfo != null) {
+                                Spacer(Modifier.width(4.dp))
+                                Image(
+                                    painter = painterResource(id = R.drawable.ic_bowling_pins),
+                                    contentDescription = "Bowling Pins",
+                                    colorFilter = ColorFilter.tint(color = GRAY04)
+                                )
+                            }
+                            if (gym.swimmingInfo != null) {
+                                Spacer(Modifier.width(4.dp))
+                                Image(
+                                    painter = painterResource(id = R.drawable.ic_swimming_pool),
+                                    contentDescription = "Bowling Pins",
+                                    colorFilter = ColorFilter.tint(color = GRAY04)
+                                )
+                            }
+                            if (gym.gymnasiumInfo != null) {
+                                Spacer(Modifier.width(4.dp))
+                                Image(
+                                    painter = painterResource(id = R.drawable.ic_basketball_hoop),
+                                    contentDescription = "Bowling Pins",
```

Comment:
same here

Reply:
Resolved in [277477d](https://github.com/cuappdev/uplift-android/pull/8/commits/277477d56e27c3145bbb6441590322fc88fc5741).